src/main/java/nitezh/ministock/activities/ChartActivity.java

```diff
@@ -1,21 +1,35 @@
 package nitezh.ministock.activities;
 
 import android.app.Activity;
+import android.app.PendingIntent;
+import android.appwidget.AppWidgetManager;
+import android.content.Context;
+import android.content.Intent;
 import android.graphics.Bitmap;
 import android.graphics.BitmapFactory;
 import android.os.AsyncTask;
 import android.os.Bundle;
 import android.text.Html;
 import android.text.Spanned;
 import android.util.Log;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
 import android.widget.ImageView;
 import android.widget.TextView;
 
+import com.google.gson.JsonObject;
+
 import java.io.InputStream;
 import java.net.URL;
+import java.util.List;
 
 import nitezh.ministock.R;
+import nitezh.ministock.WidgetProvider;
 import nitezh.ministock.activities.widget.WidgetProviderBase;
+import nitezh.ministock.activities.widget.WidgetRow;
```

Comment:
` import nitezh.ministock.activities.widget.WidgetProviderBase; import nitezh.ministock.activities.widget.WidgetRow;`

are not used

Reply:
fixed